api/room/schema_query.py

```diff
@@ -367,13 +370,19 @@ def resolve_analytics_ratios(
             self, query, start_date, end_date)
         return ratio
 
-    @Auth.user_roles('Admin', 'Default User')
+    @Auth.user_roles('Admin')
```

Comment:
hello @jesseinit, Kindly explain why this was removed. This was added so that all users are capable of viewing the analytics

Reply:
Thanks for pointing that out. I must have interpreted the story wrong.
Would update ASAP.